src/mkl_pardiso.jl

```diff
@@ -13,7 +13,7 @@ mutable struct MKLPardisoSolver <: AbstractPardisoSolver
 end
 
 function MKLPardisoSolver()
-    if !MKL_jll.is_available()
+    if !( LOCAL_MKL_FOUND || MKL_jll.is_available())
         error("MKL is not available no this platform")
```

Comment:
Maybe this error message is a bit too specific now (talking about the platform) when there could be other reasons MKL is not available